ClassWork_9/Homework_7/Homework.html

```diff
@@ -0,0 +1,180 @@
+    name : "Women's Dress Sleeveless",
+    description : " Sleeveless Cocktail Dresses till Knee Length",
+    Price: "37€" ,
+     shippingCost :{
+         cost : freeShipping
+     }
+},
+{
+    imageUrl : "https://images-na.ssl-images-amazon.com/images/I/81Yq1G6O-CL._AC_UY445_.jpg" ,
+    name : "Women's Maxi Floral dress",
+    description : " Multicolored casual Floral dress with line",
+    Price: "25€"  ,
+     shippingCost :{
+         cost : freeShipping
+     }
+},
+{
+    imageUrl : "https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRF1avcP5Nli-Mu5mYvk1HvIhqu8QpFFumFonCHNNBwmuP9_1f4Qox_dJLTiM9ep58ZFTXEcHE&usqp=CAc",
+    name : "Women's Summer dress",
+    description : " Long Casual Elegant Dresses Party Dress ",
+    Price: "35€", 
+     shippingCost :{
+         cost : freeShipping
+     }
+},
+{
+    imageUrl : "https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRMancKgpx0vgr9eGIYERMONz6vWT-F5XuMgWmBnrjbOD04KCmQ0KYi-lEVsTrPe6hb264ZLrhU&usqp=CAc",
+    name : "Women's Nightie",
+    description : " Short,pleasantly loose cut ,100% cotton",
+    Price: "15€", 
+     shippingCost :{
+         cost : freeShipping
+     }
+}
+
+] ;
+
+
+    
+
+
+    dressListCmp.innerHTML = `
+    <li>
+    <h5>${dressList[0].name}</h5>
+    <img  class ="dressImage" src="${dressList[0].imageUrl}" />
+    <p>${dressList[0].description}</p>
+    <p>Price:${dressList[0].Price}</p>
+    <span class="fa fa-star checked"></span>
```

Comment:
Can you create stars with a function. Like make dress object to have property 'rating' and add as many starts as states in the raiting.

Reply:
already did this